### DISCUSSION TITLE
Create an organization without adding the creator as a member

### DISCUSSION THREAD

**Discussion by mario-jerkovic** [clerk/javascript#4129] [Feature Requests]
Hello 👋,

I have a question and would like to discuss a particular design choice regarding the create organization action within Clerk. Specifically, I’m curious about the requirement that the createdBy field cannot be null and that the user who creates the organization must be part of it.

While I understand the importance of having a createdBy field for logging and audit purposes, I’m wondering why this user is required to be a part of the organization they’re creating. This design choice seems to enforce that the organization cannot be empty, which leads to some workarounds in our system. For example, if a user is not tied to any organization, or is part of another organization, they must still be added as an admin/member (or whatever role is defined as the creator role) in the newly created organization.

Could you provide some insight into the reasoning behind this requirement? Are there any best practices or recommendations for handling cases where a user might create an organization but shouldn’t necessarily be a part of it afterward?

Looking forward to hearing your thoughts!

Thank you!

**Comment by jescalan** [comment]
Hey there!

Sorry for the slow response - we have recently pushed out a change that does allow creation of organizations with an empty `createdBy` field. Hope this helps to resolve your issue!
